build-system/pr-check/visual-diff-tests.js

```diff
@@ -31,19 +32,24 @@ const {runCiJob} = require('./ci-job');
 const jobName = 'visual-diff-tests.js';
 
 function pushBuildWorkflow() {
+  downloadModuleOutput();
   downloadNomoduleOutput();
   timedExecOrDie('gulp update-packages');
   process.env['PERCY_TOKEN'] = atob(process.env.PERCY_TOKEN_ENCODED);
-  timedExecOrDie('gulp visual-diff --nobuild --master');
+  timedExecOrDie(`gulp visual-diff --nobuild --master --esm`);
+  timedExecOrDie(`gulp visual-diff --nobuild --master`);
 }
 
 function prBuildWorkflow() {
   process.env['PERCY_TOKEN'] = atob(process.env.PERCY_TOKEN_ENCODED);
   if (buildTargetsInclude(Targets.RUNTIME, Targets.VISUAL_DIFF)) {
+    downloadModuleOutput();
     downloadNomoduleOutput();
     timedExecOrDie('gulp update-packages');
+    timedExecOrDie('gulp visual-diff --nobuild --esm');
     timedExecOrDie('gulp visual-diff --nobuild');
   } else {
+    timedExecOrDie('gulp visual-diff --empty --esm');
     timedExecOrDie('gulp visual-diff --empty');
```

Comment:
So... this still generates two separate builds and has Percy report the 2nd one

Basically you have to loop through both module and nomodule `gulp serve` inside the visual-diff/index.js job - everything needs to run from when `launchPercyAgent()` is called and finished before `exitPercyAgent_()` - this is what starts and finalizes a build on Percy

Reply:
got it. ill make the change